docs/pages/components/tool-bar.md

```diff
@@ -1,11 +1,13 @@
 ---
 title: Toolbar
-keywords: 
+keywords:
 sidebar: left-navigation-sidebar
 toc: false
 permalink: components/toolbar.html
 folder: components
+summary: <span class="fd-badge fd-badge--error fd-badge--pill">Deprecated</span>
 ---
+> <small>`mega-menu` component has been deprecated and no longer supported </small>
```

Comment:
Should that say toolbar rather than mega-menu?

Reply:
fixed